core/src/components/input/test/basic/input.e2e.ts

```diff
@@ -63,6 +62,45 @@ configs().forEach(({ title, screenshot, config }) => {
         await expect(input).toHaveScreenshot(screenshot(`input-with-clear-button-stacked`));
       });
     });
+    test.describe('input click behaviors', () => {
+      test('should not have visual regressions', async ({ page }) => {
+        await page.setContent(
+          `
+          <ion-input
+            label="Label"
+            clear-input="true"
+            value="Text"
+          ></ion-input>
+        `,
+          config
+        );
+        const input = page.locator('ion-input');
+        input.dblclick();
+        // Validates the display of an input with a clear button.
+        await expect(input).toHaveScreenshot(screenshot(`input-dbclick`));
+      });
+      test('should not have visual regressions with stacked label', async ({ page, browserName }) => {
+        if (browserName === 'firefox') {
+          //TODO: Figure out why it's not working for firefox
+          test.skip();
+        }
+        await page.setContent(
+          `
+          <ion-input
+            label="Label"
+            label-placement="stacked"
+            clear-input="true"
+            value="Text"
+          ></ion-input>
+        `,
+          config
+        );
+        const input = page.locator('ion-input');
+        input.dblclick();
```

Comment:
I think the reason that the native input should be used to double click on is because we verify that the mouse actually is on top of the input. If we use just `ion-input` then the Playwright mouse might be clicking on the right area to trigger a focus when using specific browsers.

Also it helps to add the missing `await` to `dbclick`.

```suggestion
        const input = page.locator('ion-input');
        const nativeInput = input.locator('input');
        await nativeInput.dblclick();
```

Reply:
aah I thought we wanted to test it specifically with the ion-input but you're right, targeting the input itself makes more sense

Reply:
Sometimes we have to do stuff like this because Playwright ends up clicking the wrong area and it's a bit hard to tell where it's clicking on. By using the native input, we are removing any doubt of its location.